src/lib/components/ExpertBlock.tsx

```diff
@@ -1,15 +1,24 @@
 import React from 'react';
+import { useStyles } from '../../modules/experts/styles/ExpertBlock.style';
 import { IExpert } from '../types';
 
 interface IExpertProps {
   expert: IExpert;
+  nameClass: string;
```

Comment:
see another comment, it should not be here

Reply:
deleted